dev/breeze/tests/test_selective_checks.py

```diff
@@ -676,7 +700,7 @@ def test_expected_output_full_tests_needed(
         ),
     ],
 )
-def test_expected_output_pull_request_v2_3(
+def test_expected_output_pull_request_v2_7(
```

Comment:
It's unrelated, but I noticed that we have not updated the branch to latest - it does not matter, but might be misleading